.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+ name: Feature/Improvement request
+ about: Request a feature or improvement
+ title: ''
+ labels: 'enhancement, moonshot'
+ assignees: ''
+-
+2---
```

Comment:
There's something wrong with this file I think? Does it contain these `2`s?

Reply:
wut ! this should be only in my vim arf!